index.html

```diff
@@ -120,7 +120,32 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-
+<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+<div class="card">
+  <p class="name">Margaux Hutchins</p>
+  <p class="contact">
+    <i class="fab fa-twitter"></i>
+    <a href="https://twitter.com/HutchinsMargaux" target="_blank">@HutchinsMargaux</a>
+    <i class="fab fa-github"></i>
+    <a href="https://www.github.com" target="_blank">Your handle</a>
```

Comment:
Hi, you did well in creating your card, but you should also update your github handle like below.
You can update this PR by committing and pushing again, then I will review it.
Thank you for your contribution.

```suggestion
    <i class="fab fa-github"></i>
    <a href="https://www.github.com/mhutchins78" target="_blank">mhutchins78</a>
```